relay-server/src/actors/project.rs

```diff
@@ -811,22 +795,25 @@ impl Handler<GetProjectState> for Project {
 ///  - Cached rate limits
 #[derive(Debug)]
 pub struct CheckEnvelope {
-    envelope: Envelope,
-    fetch: bool,
+    pub public_key: ProjectKey,
+    pub envelope: Envelope,
+    pub fetch: bool,
```

Comment:
Please keep these attributes all private. Except for legacy messages, we should no longer be leaking internal attributes to public interfaces.

Reply:
done